GNUmakefile

```diff
@@ -0,0 +1,71 @@
+# Makefile for the supdup server and client.
```

Comment:
Please keep the old file name.

Reply:
Any reason why?

The Makefile does (and did before my changes as well) require GNU Make, which always looks for GNUmakefile first, and falls back to Makefile or makefile for compatibility.

Leaving it as is works, but you'll a lot of unhelpful error messages trying to build with the default make of Solaris or the BSDs without any indication that GNU Make is required (without reading the README, I guess).

Reply:
You'll get a similar unhelpful error message even if you rename it to GNUmakefile, or worse do "make supdup" it will try to compile supdup anyway, and most likely bail hard.

Reply:
Well, “`no target to make`”, IMO, is better than:
```sh
make: "/home/jhj/src/supdup/GNUmakefile" line 14: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 16: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 19: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 21: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 24: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 27: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 30: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 32: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 35: Invalid line type
make: "/home/jhj/src/supdup/GNUmakefile" line 38: Invalid line type
make: Fatal errors encountered -- cannot continue
```

I could also add a POSIX/BSD-compatible Makefile that attempts to invoke `gmake` if it is available in the PATH, and provide a friendlier error if not.

Reply:
The reason is that I'm generally wary of cosmetic changes.  In my opinion, they are often not worth it.